### PR TITLE
[JBPM-9237] Test queries with columns 'PROCESS_INST_ID', 'ACTIVITY_ID'

### PR DESCRIPTION
**Pull request by gmunozfe**
for case-sensitive dbs

### CODE REVIEW

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/QueryDataServiceIntegrationTest.java [hunk 1]
```diff
@@ -1051,6 +1052,32 @@ public void testQueryDataServiceUsingComplexCustomQueryBuilderFilterSpecWithOrde
 
 
     }
+    
+    @Test
+    public void testDefaultQueryJbpmProcessInstances() throws Exception {
```
Comment (afalhambra) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/QueryDataServiceIntegrationTest.java:1057]: Doesn't really throw any exception inside this method.

```suggestion
    public void testDefaultQueryJbpmProcessInstances() {
```

**Comment by gmunozfe** [comment]
> looking good for me, but how to test it on all kinds of databases? (how to set case sensitivity database?)

As datasource is parameterized, then each instance in jenkins reallocates a different database by means of dballocator.
Regarding case sensitivity, I think it's out of our control to change collation at dballocator, but at least we can test those queries in the certified environments.

**Comment by bxf12315** [comment]
> dballocator

I can not find case-sensitivity database of mssql. So I pull mssql docker image to test it. case-sensitivity not default settings of mssql.

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/QueryDataServiceIntegrationTest.java (hunk 1 above)
Comment (gmunozfe) [reply kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/QueryDataServiceIntegrationTest.java]: done

**Pull request merged**